src/frontend/apps/lti_site/components/ErrorComponents/BoundaryScreenError/index.tsx

```diff
@@ -0,0 +1,148 @@
+import { Box, Paragraph, Image } from 'grommet';
+import React from 'react';
+import { useIntl, FormattedMessage } from 'react-intl';
+
+import { H2 } from 'components/Headings';
+import { LayoutMainArea } from 'components/LayoutMainArea';
+import { useAppConfig } from 'data/stores/useAppConfig';
+
+const messages = {
+  altLogo: {
+    defaultMessage: 'Marsha logo',
+    description:
+      'Accessible description for the Marsha logo displayed in the error page',
+    id: 'components.ErrorComponents.altLogo',
+  },
+  altErrorDrawing: {
+    defaultMessage:
+      'Error drawing of a person searching something from a telescope',
+    description:
+      'Accessible description for the Error drawing of a person searching something from a telescope',
+    id: 'components.ErrorComponents.altErrorDrawing',
+  },
+  onomatopoeia: {
+    defaultMessage: 'Ooops !',
+    description:
+      'Onomatopoeia of a person saying "Ooops !" in a surprised tone',
+    id: 'components.ErrorComponents.onomatopoeia',
+  },
+  introduceProblem: {
+    defaultMessage: 'There seems to be a slight problem:',
+    description: 'Sentence to introduce the error message in the error page',
+    id: 'components.ErrorComponents.probleme',
+  },
+};
+interface IBoundaryScreenErrorProps {
+  code: number;
+  message: string;
+}
+export const BoundaryScreenError = ({
+  code,
+  message,
+}: IBoundaryScreenErrorProps) => {
+  const appData = useAppConfig();
+  const intl = useIntl();
+
+  return (
+    <LayoutMainArea
+      style={{
+        display: 'flex',
+        justifyContent: 'center',
+      }}
+    >
+      <div
+        style={{
+          display: 'flex',
+          height: '100vh',
+          width: '100vw',
+          gap: '1rem',
+          backgroundColor: '#f8fafe',
```

Comment:
And uses colors from the theme (or add those in the theme if they are not already in it)